pkg/relayer/proxy/relay_verifier.go

```diff
@@ -3,8 +3,10 @@ package proxy
 import (
 	"context"
 
+	sdkerrors "cosmossdk.io/errors"
+	ring_secp256k1 "github.com/athanorlabs/go-dleq/secp256k1"
 	"github.com/cometbft/cometbft/crypto"
-	accounttypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/noot/ring-go"
```

Comment:
Just FYI that this is finally happening @noot  :)

Reply:
Big up @noot loving the ring library <3